doc/content/modules/installation-guide/pages/how-tos/install.adoc

```diff
@@ -3,157 +3,9 @@
 This section delves into the detailed step-by-step instructions for installing {product}.
 Detailed explanations walk you through each step.
 
-== Get {product}
+{product} can be installed following several different patterns :
```

Comment:
In english, double punctuations don't need a space before.
Could you fix that here and everywhere else in this PR please?